app/models/resource.py

```diff
@@ -165,10 +165,13 @@ def generate_fake(count=20, center_lat=39.951021, center_long=-75.197243):
     @staticmethod
     def get_resources_as_dicts(resources):
         resources_as_dicts = [resource.__dict__ for resource in resources]
```

Comment:
i'm a little confused here - you put in the `resource.__dict__` for every resource but then also append each resource's dictionary afterward as well with the `avg_rating` so wouldn't that be two dictionaries for each resource (one with and one without average rating)?
I think you should be able to just initialize this to `[]`?